components/wsdl-tools/wsdl2code/org.wso2.carbon.modules.wsdl2code.ui/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>org.wso2.carbon.commons</groupId>
             <artifactId>org.wso2.carbon.wsdl2code.stub</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.org.owasp.encoder</groupId>
+            <artifactId>encoder</artifactId>
+        </dependency>
```

Comment:
Since this is applied to future releases of the products it will not cause any issue